libs/test/SmartSyncTest/src/com/salesforce/androidsdk/smartsync/manager/ManagerTestCase.java

```diff
@@ -131,5 +144,68 @@ private RestClient initRestClient() throws Exception {
                 null, null, null, null, null, null);
         return new RestClient(clientInfo, authToken, httpAccess, null);
     }
-    
+
+    /**
+     * Helper methods to create "count" of test records
+     * @param count
+     * @return map of id to name for the created accounts
+     * @throws Exception
+     */
+    protected Map<String, String> createRecordsOnServer(int count, String objectType) throws Exception {
+        Map<String, String> idToValues = new HashMap<String, String>();
+        for (int i = 0; i < count; i++) {
+
+            // Request.
+            String fieldValue = createRecordName(objectType);
+            Map<String, Object> fields = new HashMap<String, Object>();
+            //add more object type if need to support to use this API
+            //to create a new record on server
+            switch (objectType) {
+                case Constants.ACCOUNT:
+                    fields.put(Constants.NAME, fieldValue);
```

Comment:
for some type of record, like Case, it does not have Name field, that's why I leave it inside of the switch block.